### ISSUE TITLE
Replays fail to record correctly using new event flairs

### ISSUE THREAD

**Issue by ballparts** [chrahunt/TagProReplays#214]
From [this reddit post](https://www.reddit.com/r/TagPro/comments/7yrtd6/tagpro_replays_not_showing/)

Player data appears to fail to record (all arrays appear to be full of `null` values) when a player is wearing one of the new flairs. 

I recreated the issue by joining a game on origin (only player in the game) wearing the coin flair. By coincidence the map was plasma, the same map in the user's raw replay file. Just in case that is related :)

**Comment by ballparts** [comment]
Nothing looks particularly strange about the flair object in-game.
![screen shot 2018-02-20 at 2 55 45 pm](https://user-images.githubusercontent.com/7952062/36446089-3a4a35f2-164e-11e8-8303-0565cbac8a9b.png)

**Comment by ballparts** [comment]
Well, it looks like I was mistaken and I introduced a red herring. I had previously tried to import the replay the user uploaded into my extension, and I interpreted further errors as having to do with the new flairs. The extension was simply producing errors while attempting to parse that user's replay as opposed to the new ones I recorded. 

So, here is the information I know at this point:

* The replay uploaded from the user has several player objects full of arrays of `null` values. 

My interpretation is that the extension removed players with all null values prior to saving, and since the recording player was one of the players with `null` values, the replay no longer had a player with `me` equal to "me". 

So I still don't know what caused the extension to fail to properly record the user's data. There are 16 total players in that replay, but it is possible that player exited and re-entered the game several times. It might be that many or all of the `null` players represent the recording player. 

I will follow up with the player. 

This issue's title is now misleading, so I will close it and open a new one.
